### PR TITLE
Improve project readme file

### PR DESCRIPTION
**Pull request by djelinek**
Signed-off-by: Dominik Jelinek <djelinek@redhat.com>

### CODE REVIEW

File: FAQ.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Frequently Asked Questions
+
+## Tests getting stuck on launch on Linux (CentOS based)
```
Comment (odockal) [inline FAQ.md:3]: Maybe known issues is better description of this part?

File: README.md [hunk 2]
```diff
@@ -1,96 +1,75 @@
...
+- Download a test instance of Visual Studio Code
+- Download the appropriate version of ChromeDriver
+- Package and Install your extension into downloaded VS Code instance
+- Launch the VS Code instance using Selenium Webdriver
+- Run your tests
+
+### Requirements
+ - **Visual Studio Code 1.37+**
+ - **NodeJS 16**
+ - **JDK 11+**
```
Comment (odockal) [inline README.md:41]: Java?

Comment (djelinek) [reply README.md]: ops.. not strictly required.. I will remove that one.. nice catch! 👍

**Review by ssbarnea — CHANGES_REQUESTED** [review]
This is not valid markdown, I would recommend that before adding these to enable a linter/formatter for markdown.

Markdown does not use spacing before paragraphs.

**Comment by djelinek** [comment]
> This is not valid markdown, I would recommend that before adding these to enable a linter/formatter for markdown.
> 
> Markdown does not use spacing before paragraphs.

@ssbarnea I fixed the indents in known issues markdown file.

**Pull request merged**